### PR TITLE
Add validation in other gitlab instances

### PR DESCRIPTION
**Pull request by majamassarini**
Validation script for:
- gitlab.gnome.org
- gitlab.freedesktop.org
- salsa.debian.org

Merge before: packit/helm#23

### CODE REVIEW

**Comment by majamassarini** [comment]
reheck

**Comment by majamassarini** [comment]
recheck

File: cron-jobs/packit-service-validation/packit-service-validation.py [hunk 1]
```diff
@@ -661,9 +661,14 @@ def run(self):
...
+        instance_url="https://gitlab.com",
+        namespace="packit-service",
+        token_name="GITLAB_TOKEN",
+    ):
         gitlab_service = GitlabService(
-            token=getenv("GITLAB_TOKEN"), instance_url=instance_url
+            token=getenv(token_name), instance_url=instance_url
         )
         self.project: GitlabProject = gitlab_service.get_project(
             repo="hello-world", namespace="packit-service"
```
Comment (lbarcziova) [inline cron-jobs/packit-service-validation/packit-service-validation.py:674]:
```suggestion
            repo="hello-world", namespace=namespace
```

**Review by lbarcziova — APPROVED** [review]

File: cron-jobs/packit-service-validation/packit-service-validation.py (hunk 1 above)
Comment (majamassarini) [reply cron-jobs/packit-service-validation/packit-service-validation.py]: 🙏

**Pull request merged**